### PR TITLE
feat: lp ammendment test

### PR DESCRIPTION
**Pull request by bwallacee**
Asserting against the grid could do with a refactor but I think in the future we need a function for this in general so I have got this in quickly for now.

### CODE REVIEW

File: tests/liquidity_provision/test_liquidity_provision.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+def continuous_market(vega):
+    return setup_continuous_market(vega)
+
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_liquidity_provision_amendment(continuous_market, vega: VegaService, page: Page):
+    # TODO Refactor asserting the grid
+    page.goto(f"/#/liquidity/{continuous_market}")
+    page.get_by_test_id("manage-vega-wallet").click()
+    page.locator('[role="menuitemradio"] >> .mr-2.uppercase').nth(1).click(position={ "x": 0, "y": 0}, force=True)
```
Comment (dalebennett1992) [inline tests/liquidity_provision/test_liquidity_provision.py:26]: what's this for? I always wince a bit when I see something clicked with co-ordinates. Is there a better way to do this?

File: tests/liquidity_provision/test_liquidity_provision.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    page.reload()
+    row = page.get_by_test_id("tab-myLP").locator(".ag-center-cols-container .ag-row").first
+    expect(row).to_contain_text(
+        "Active10,000.0010,000.000.00%10,000.00100.00%10,000.00-100.00%0.00%---5/9/2023,"
```
Comment (dalebennett1992) [inline tests/liquidity_provision/test_liquidity_provision.py]: What dictates this date here? Are we setting this somewhere?

File: tests/liquidity_provision/test_liquidity_provision.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    )
+    vega.submit_simple_liquidity(
+        key_name="mm",
+        market_id=continuous_market,
+        commitment_amount=100,
+        fee=0.001,
+        is_amendment=True,
+    )
+
+    vega.wait_fn(1)
```
Comment (dalebennett1992) [inline tests/liquidity_provision/test_liquidity_provision.py:40]: not actionable but another item to address is potentially having a wait lib where we can wait reliably.

**Review by dalebennett1992 — APPROVED** [review]
with suggestions and some comments. I will let you use your judgement as to whether it goes through.

File: tests/liquidity_provision/test_liquidity_provision.py (hunk 2 above)
Comment (bwallacee) [reply tests/liquidity_provision/test_liquidity_provision.py]: Removing the date part, but yeah it is hardcoded in

File: tests/liquidity_provision/test_liquidity_provision.py (hunk 1 above)
Comment (bwallacee) [reply tests/liquidity_provision/test_liquidity_provision.py:26]: Playwright by default will click the middle of an element
annoyingly for us that means clicking the copy button so they it doesn't actually click what we want it to:
image
Can fix this in the future by renaming the wallet from "mm" to "marketMaker" but I don't want to do that yet just incase it breaks loads of tests'
I'll add a comment explaining why and a TODO

**Pull request merged**